src/test/java/com/wikia/webdriver/elements/oasis/components/navigation/NestedNavigation.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.elements.oasis.components.navigation;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+
+public class NestedNavigation extends BasePageObject {
+
+  @FindBy(xpath = "/html/body/div[3]/header/nav/ul/li[2]/div")
+  private WebElement communityButton;
+  @FindBy(xpath = "/html/body/div[3]/header/nav/ul/li[2]/div/div[2]/ul/li[3]")
+  private WebElement scrollTest3Button;
+  @FindBy(xpath = "/html/body/div[3]/header/nav/ul/li[2]/div/div[2]/ul/li[3]/div/ul/li[8]")
+  private WebElement cat_3Button;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "cat_3Button" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)